camus-shopify/script/fab

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import os
+from fabric.api import env
+from fabric.operations import run, put
+
+env.forward_agent = True
+env.hosts = ['spark-etl1.chi.shopify.com']
+env.user = 'deploy'
+app_name = 'camus'
+code_dir = '/u/apps/camus/current'
+path = os.path.join(path, 'camus-shopify', 'target', 'camus-shopify-0.1.0-shopify1.jar')
+
+run('mkdir -p %s' % code_dir)
```

Comment:
Chef already sets up /u/apps/Camus
We only create the current repo which if I understand is fab or cap
responsibility

On Thursday, April 9, 2015, Harry Brundage <notifications@github.com> wrote:

> In camus-shopify/script/fab
> <https://github.com/Shopify/camus/pull/36#discussion_r28063943>:
>
> > @@ -0,0 +1,15 @@
> > +#!/usr/bin/env python
> > +import os
> > +from fabric.api import env
> > +from fabric.operations import run, put
> > +
> > +env.forward_agent = True
> > +env.hosts = ['spark-etl1.chi.shopify.com']
> > +env.user = 'deploy'
> > +app_name = 'camus'
> > +code_dir = '/u/apps/camus/current'
> > +path = os.path.join(path, 'camus-shopify', 'target', 'camus-shopify-0.1.0-shopify1.jar')
> > +
> > +run('mkdir -p %s' % code_dir)
>
> This won't work because the deploy user doesn't and shouldn't have
> permission to create dirs in /u/apps. Really, this just isn't standard,
> you need to use chef to set up the deploy environment using the app_setup
> helper. See the grid or the starscream recipes :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Shopify/camus/pull/36/files#r28063943>.
>